tests/foreman/ui/test_subscription.py

```diff
@@ -618,6 +618,7 @@ def test_positive_check_manifest_validity_notification(
         target_sat.upload_manifest(function_org.id, function_sca_manifest.content)
 
         # Initially 'Manifest expiring soon' message not found
+        assert not session.subscription.is_subscription_manifest_header_message_display()
```

Comment:
I would suggest to handle these messages on airgun side instead of here. Assert the `expire` message in your related airgun PR.